app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
```

Comment:
You could indicate in the type annotation that arguments that have default values are optional in this way
```suggestion
            health: int | None = 100,
```